astropy/io/fits/scripts/fitsdiff.py

```diff
@@ -247,12 +261,25 @@ def main():
 
     opts, args = handle_options(argv)
 
+    if opts.tolerance is not None and opts.rtol is not None:
```

Comment:
Yeah, maybe it's better to issue an `AstropyDeprecationWarning`  and let `tolerance` overwrite `rtol`